sdk/ml/azure-ai-ml/tests/component/unittests/test_command_component_entity.py

```diff
@@ -22,6 +23,19 @@
 from .._util import _COMPONENT_TIMEOUT_SECOND
 
 
+@pytest.fixture()
+def enable_early_available_output() -> None:
```

Comment:
may change a different name, this fixture actually does the schema reload not only for early available

Reply:
updated to `enable_private_preview_schema_features`, same as Azure DevOps comment